scripts/benchmark.cr

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env -S crystal run --no-debug --release
+
+require "../src/pluto"
+
+record BenchmarkResult, name : String, time : Int64, memory : Int64
+
+def benchmark_memory
+  bytes_before_measure = GC.stats.total_bytes
+  yield
+  (GC.stats.total_bytes - bytes_before_measure).to_i64
+end
+
+def benchmark_time
+  time = Time.monotonic
+  yield
+  Time.monotonic - time
+end
+
+def print_result_table(benchmarks : Array(BenchmarkResult))
+  name_rjust = benchmarks.map(&.name.size).max
+  time_ljust = benchmarks.map { |result| result.time.to_s.size + 2 }.max
```

Comment:
I believe this can be `benchmarks.map(&.time.to_s.size.+(2)).max`

Reply:
Much better! I forgot that operators are just methods for a minute :P